### PR TITLE
Stability reference for #4288

### PR DESCRIPTION
**Pull request by kalikiana**
Whilst #4352 doesn't work I'm opening this to provide a reference for the stability of #4288.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -177,7 +177,7 @@ test-unstable:
 
 .PHONY: test-fullstack
 test-fullstack:
-	$(MAKE) test-with-database FULLSTACK=1 TIMEOUT_M=30 RETRY=15 PROVE_ARGS="$$HARNESS t/full-stack.t"
+	$(MAKE) test-with-database FULLSTACK=1 TIMEOUT_M=30 RETRY=50 STABILITY_TEST=1 PROVE_ARGS="$$HARNESS t/full-stack.t"
```
Comment (Martchus) [inline Makefile:180]: You'll likely also need to increase the timeout.

**Comment by kalikiana** [comment]
For the record, `Generate packaged assets` says `Package 'perl-Mojolicious-Plugin-OAuth2-1.59' not found.`.

**Comment by kalikiana** [comment]
See [](https://progress.opensuse.org/issues/102332#note-2) regarding tests failing with `unable to read [..]/00000001-tinycore-1-flavor-i386-Build1-core@coolone/autoinst-log.txt: Can't open file "[...]/00000/00000001-tinycore-1-flavor-i386-Build1-core@coolone/autoinst-log.txt": No such file or directory at t/full-stack.t line 161.`

**Comment by kalikiana** [comment]
I don't want to maintain two PRs anymore since I'm getting confused where I did and didn't make changes. Hence using the original for stability now

**Pull request closed without merging**